test/enterprise-integration-test-values.yaml

```diff
@@ -150,23 +150,10 @@ alfresco-sync-service:
       cpu: "2"
       memory: "1Gi"
   postgresql: *postgresql-sync
-alfresco-ai-transformer:
-  enabled: true
-  replicaCount: 1
-  aws:
-    accessKeyId: AAAAAAAAAAAAAAAAA
-    secretAccessKey: ZZZZZZZZZZZZZZZZZ
-    s3Bucket: somebucket
-    region: us-east-1
-    comprehendRoleARN: arn:aws:iam::000000000000:user/comprehend
-  resources:
-    requests:
-      cpu: "0.01"
-      memory: "512Mi"
```

Comment:
in this way we are not testing it anymore in acs-deployment

Reply:
True but this test do not add any value compared to the component chart's tests we run with `ct`

Reply:
I guess I removed it in an attempt to speed up deployment (before I enabled the sequential logging during installation). If you're really sure we can catch something here I can try to re-enable it (though I'd like to know what we can catch here tbh)

Reply:
e.g. if we are injecting configmaps in a consistent manner (with the appropriate keys) - which are managed in this chart only